### PR TITLE
MWPW-131707 Get correct region for caas autolang

### PR DESCRIPTION
**Pull request by chrischrischris**
* Correctly determine country for caas auto-country/lang

Resolves: [MWPW-131707](https://jira.corp.adobe.com/browse/MWPW-131707)

**Test URLs:**
- Bacom Before:
  - No Thai cards displayed
  - https://main--bacom--adobecom.hlx.page/th_th/resources/main
- Bacom After:
	- Thai cards displayed	
	- https://main--bacom--adobecom.hlx.page/th_th/resources/main?milolibs=caaslocale
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://caaslocale--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Review by Ruchika4 — APPROVED** [review]

**Review by mokimo — APPROVED** [review]

**Comment by mokimo** [comment]
FYI a caas test is failing, that needs to be adapted before merging

File: libs/utils/utils.js [hunk 1]
```diff
@@ -146,6 +146,7 @@ export function getLocale(locales, pathname = window.location.pathname) {
     return locale;
   }
   locale.prefix = locale.ietf === 'en-US' ? '' : `/${localeString}`;
+  [locale.region] = localeString.split('_');
```
Comment (overmyheadandbody) [inline libs/utils/utils.js:149]: Does this cover all use-cases? Say we have these pathname examples:

1. `/products/photoshop`
2. `/ch_de/products/photoshop`
3. `/ro/products/photoshop`
4. `/`

Looking at the code, the full formula for `locale.region` seems to be `pathname.split('/')[1].split('_')[0]`. For the use-cases above, the results would be:

1. `'products'`
2. `'ch'`
3. `'ro'`
4. `''`

So to me it seems that examples 2 & 3 are working as expected, but I'm not so sure about 1 & 4. 4 might be ok for the purposes of `caas/utils.js`, but 1 still stands out. What would be the desired outcome?

**Review by overmyheadandbody — COMMENTED** [review]
I second @mokimo 's observation around unit tests and have a quick question.

File: libs/utils/utils.js (hunk 1 above)
Comment (chrischrischris) [reply libs/utils/utils.js]: Thanks for catching that.  Fixed.

**Pull request merged**